Makefile

```diff
@@ -42,55 +42,121 @@ compile:
 	@cd ../java
 	@make
 	@cd ../eo/
-	@eoc dataize app --verbose
 run:
 	@clear
-	@java -cp targets/java/ GraphGenerator 0 10 > tests/edges/star3.graph
-	@java -cp targets/java/ GraphGenerator 1 10 > tests/list/dijkstra2.graph
 	@cd tests/edges
 	@echo "Now we are going to run Kruskal's algorithm \n";
-	@for FILE in *; do \
+	@for FILE in *.graph; do \
 	    var=$$(cat $$FILE)
 		echo "Test ($$FILE) is running"; \
 		$(J); printjava $$(java -cp $(TARGPATH)/java Kruskal $$var); \
 		$(C); printcpp $$($(TARGPATH)/cpp/kruskal $$var); \
+		
+	@cd ../../src/eo/kruskal
+		# $(E); printeo @$$(eoc --easy dataize kruskalApp $$var)
+	@cd ../../../tests/edges/
 		echo "\n";
 	done
 
 	@echo "Now we are going to run Prim's algorithm \n";
-	@for FILE in *; do \
+	@for FILE in *.graph; do \
 	    var=$$(cat $$FILE)
 		echo "Test ($$FILE) is running"; \
 		$(J); printjava $$(java -cp $(TARGPATH)/java prim/PrimMST $$var); \
 		$(C); printcpp $$($(TARGPATH)/cpp/prim $$var); \
 
-	@cd ../../src/eo
-		$(E); printeo $$(eoc --alone dataize app $$var)
-	@cd ../../tests/edges/
+	@cd ../../src/eo/prim
+		# $(E); printeo @$$(eoc --easy dataize primApp $$var)
+	@cd ../../../tests/edges/
 		echo "\n";
 	done
 
 	@cd ../list
 	@echo "Now we are going to run Dijkstra's algorithm \n";
-	@for FILE in *; do \
+	@for FILE in *.graph; do \
 	    var=$$(cat $$FILE)
 		echo "Test ($$FILE) is running"; \
 		$(J); printjava $$(java -cp $(TARGPATH)/java Dijkstra $$var); \
 		$(C); printcpp $$($(TARGPATH)/cpp/dijkstra $$var); \
+	
+	@cd ../../src/eo/dijkstra
+		# $(E); printeo @$$(eoc --easy dataize dijkstraApp $$var)
+	@cd ../../../tests/list/
 		echo "\n";
 	done
 
-	@echo "Now we are going to run Ford-Falkerson algorithm \n";
-	@for FILE in *; do \
+	@echo "Now we are going to run Ford-Fulkerson algorithm \n";
+	@for FILE in *.graph; do \
 	    var=$$(cat $$FILE)
 		echo "Test ($$FILE) is running"; \
 		$(J); printjava $$(java -cp $(TARGPATH)/java FordFalkerson $$var); \
 		$(C); printcpp $$($(TARGPATH)/cpp/fordfalkersonalg $$var); \
+		
+	# Ford-Fulkerson doesn't work properly
+	@cd ../../src/eo/fordfulkerson
+		# $(E); printeo @$$(eoc --easy dataize fordfulkersonApp $$var)
+	@cd ../../../tests/list/
+		echo "\n";
+	done
+
+prim:
+	@echo "Now we are going to run Prim's algorithm \n";
+	@cd tests/edges/
+	@for FILE in *.graph; do \
+	    var=$$(cat $$FILE)
+		echo "Test ($$FILE) is running"; \
+

```

Comment:
@MaxFunProger empty lines are also not helpful, but mostly hurtful: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html